__tests__/test-utils/mock-data/manager.js

```diff
@@ -13,6 +13,23 @@ import type {
 
 let COUNTER = 0
 
+/**
+ * Make a mock deployment summary given a project and some FeedVersions. This is a
+ * function so that circular references can be defined.
```

Comment:
Is there a reason the function args are not used?

Reply:
Good catch thanks! Parameters have been removed